webpack.proxyPortal.js

```diff
@@ -9,18 +9,29 @@
 const path = require('path');
 
 module.exports = () => ({
-	mode: 'development',
-	entry: path.resolve(__dirname, 'src/loader/bootstrap.js'),
-	output: {
-		path: path.resolve(__dirname, 'build/proxyPortal'),
-		filename: 'loader.js',
+	devServer: {
+		contentBase: path.join(__dirname, 'build/proxyPortal'),
+		port: 9090,
+		proxy: {
+			'/o/frontend-js-web/loader/loader.js': {
+				pathRewrite: {'^/o/frontend-js-web/loader': ''},
+				target: 'http://localhost:9090',
+			},
+
+			// eslint-disable-next-line sort-keys
```

Comment:
Here's the other place I disabled it, just in case webpack depends on object order here (I don't know that it does, but it seems like a reasonable precaution to take).